packs/zookeeper/config.yaml

```diff
@@ -0,0 +1,3 @@
+---
+zookeeper_hosts: "localhost:2181"
+zookeeper_root: "/"
```

Comment:
Yeah, I'd actually prefer to namespace it like /stackstorm/leases/ but I think anyone familiar with ZK can configure it the way they want.